units/host_monitoring_idle_termination.go

```diff
@@ -221,10 +221,17 @@ func (j *idleHostJob) checkAndTerminateHost(ctx context.Context, h *host.Host) e
 	}
 
 	// if we haven't heard from the host or it's been idle for longer than the cutoff, we should terminate
-	if communicationTime >= idleThreshold || idleTime >= idleThreshold {
+	terminateReason := ""
+
+	if communicationTime >= idleThreshold {
```

Comment:
would it be helpful to log the reason to splunk too? This reason just ends up in the db/UI, I think.
I guess it depends: why is it important to know which of these is the reason?

Reply:
I had just assumed this was logged to splunk. Will make that change.